test-cases/longevity/longevity-harry-2h.yaml

```diff
@@ -9,7 +9,7 @@ instance_type_db: 'i3.large'
 
 aws_root_disk_size_loader: 80 # enlarge loader disk, cause of cassandra-harry operation.log that can't be disabled
 
-nemesis_class_name: 'SisyphusMonkey'
+nemesis_class_name: 'NoOpMonkey'
```

Comment:
Is it for your testing?

Reply:
I'm not sure the this is ready for being run with a nemesis yet, we have issue with crashing see https://github.com/scylladb/scylla/issues/10553